docs/resources/sidecar_listener.md

```diff
@@ -47,15 +47,28 @@ resource "cyral_sidecar_listener" "listener" {
   }
 }
 
+# Listener for S3 browser (using port 444 assuming port 443 is used for CLI)
+resource "cyral_sidecar_listener" "listener_s3_cli" {
+  sidecar_id = cyral_sidecar.sidecar.id
+  repo_types = ["s3"]
+  network_address {
+    port = 444
+  }
+  s3_settings {
+    proxy_mode = false // may be omitted for s3 browser as it defaults to `false`
```

Comment:
IIRC, the problem these guys were having was because they had the "browser" listener, but not the other one, meaning the other one (proxy?) is mandatory for S3, right? should we add a comment to that effect?

Reply:
I don't think we should add extensive docs to the example as the database-specific blocks are *much* better documented now, specially this `proxy_mode` attribute.